projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -1,19 +1,24 @@
 <ng-container
   *ngIf="merchandisingCarouselModel$ | async as merchandisingCarouselModel"
 >
-  <div
-    class="data-cx-merchandising-carousel"
-    [cxAttributes]="merchandisingCarouselModel.metadata"
-    [cxAttributesNamePrefix]="'data-cx-merchandising-carousel'"
-  ></div>
-  <cx-carousel
-    [ngStyle]="carouselStyle$ | async"
-    [items]="merchandisingCarouselModel.items$"
-    [title]="title$ | async"
-    [template]="carouselItem"
-    itemWidth="285px"
-  >
-  </cx-carousel>
+  <ng-container *ngIf="backgroundColor$ | async as backgroundColor">
+    <ng-container *ngIf="textColor$ | async as textColor">
+      {{ setBackgroundColor(backgroundColor) }}
+      {{ setTextColor(textColor) }}
+      <div
+        class="data-cx-merchandising-carousel"
+        [cxAttributes]="merchandisingCarouselModel.metadata"
+        [cxAttributesNamePrefix]="'data-cx-merchandising-carousel'"
+      ></div>
+      <cx-carousel
+        [items]="merchandisingCarouselModel.items$"
+        [title]="title$ | async"
```

Comment:
we might want to use a single observable for all content, then use the `[attr.title]` to avoid a title attribute if there isn't any content.